runtime/lib/validator/MaxValueValidator.php

```diff
@@ -38,7 +38,7 @@ class MaxValueValidator implements BasicValidator
      */
     public function isValid(ValidatorMap $map, $value)
     {
-        if (is_null($value) == false && is_numeric($value) == true) {
+        if (is_null($value) === false && is_numeric($value) === true) {
             return intval($value) <= intval($map->getValue());
```

Comment:
`is_null($value) === false` why not use `null !== $value`, result will the same, less code :)
:+1:  on the numeric check :)